### DISCUSSION TITLE
Using Deepness plugin in QGIS Graphical Modeler

### DISCUSSION THREAD

**Discussion by kuncda** [PUTvision/qgis-plugin-deepness#162] [Q&A]
Is it possible to use Deepness in QGIS Graphical Modeler? Thank you for answers

**Comment by przemyslaw-aszkowski** [comment]
Hi, I don't think so, as I hear about Graphical Modeler for the first time ;) Though If there are many interested people we can investigate it.
Cheers
Przemek
